src/main/java/com/renatusnetwork/momentum/gameplay/listeners/InteractListener.java

```diff
@@ -27,20 +30,17 @@ public class InteractListener implements Listener {
     private HashMap<String, BukkitTask> resetConfirmMap = new HashMap<>();
     private HashMap<String, BukkitTask> spawnConfirmMap = new HashMap<>();
 
-    @EventHandler (priority = EventPriority.LOWEST)
-    public void onInteract(PlayerInteractEvent event)
-    {
+    @EventHandler(priority = EventPriority.LOWEST)
+    public void onInteract(PlayerInteractEvent event) {
```

Comment:
dont know why all the braces got moved to the same line; i mightve accidentally reformatted this file in intellij. i can change it back if needed